examples/terraform-gcp-example/README.md

```diff
@@ -0,0 +1,41 @@
+# Terraform GCP Example
+
+This folder contains a simple Terraform module that deploys resources in [GCP](https://cloud.google.com/) to demonstrate
+how you can use Terratest to write automated tests for your GCP Terraform code. This module deploys an [Compute
+Instance](https://cloud.google.com/compute/) and gives that Instance a `Name` tag with the value specified in the
+`instance_name` variable.
```

Comment:
Make sure to update the description of what this example deploys once it's done.